device/celestica/x86_64-cel_seastone-r0/sonic_platform/chassis.py

```diff
@@ -118,35 +118,38 @@ def get_reboot_cause(self):
             one of the predefined strings in this class. If the first string
             is "REBOOT_CAUSE_HARDWARE_OTHER", the second string can be used
             to pass a description of the reboot cause.
-        """
-        description = 'None'
-        reboot_cause = self.REBOOT_CAUSE_HARDWARE_OTHER
-
-        reboot_cause_path = (HOST_REBOOT_CAUSE_PATH + REBOOT_CAUSE_FILE) if self.is_host else PMON_REBOOT_CAUSE_PATH + REBOOT_CAUSE_FILE
-        prev_reboot_cause_path = (HOST_REBOOT_CAUSE_PATH + PREV_REBOOT_CAUSE_FILE) if self.is_host else PMON_REBOOT_CAUSE_PATH + PREV_REBOOT_CAUSE_FILE
 
-        hw_reboot_cause = self._component_list[0].get_register_value(RESET_REGISTER)
+            REBOOT_CAUSE_POWER_LOSS = "Power Loss"
+            REBOOT_CAUSE_THERMAL_OVERLOAD_CPU = "Thermal Overload: CPU"
+            REBOOT_CAUSE_THERMAL_OVERLOAD_ASIC = "Thermal Overload: ASIC"
+            REBOOT_CAUSE_THERMAL_OVERLOAD_OTHER = "Thermal Overload: Other"
+            REBOOT_CAUSE_INSUFFICIENT_FAN_SPEED = "Insufficient Fan Speed"
+            REBOOT_CAUSE_WATCHDOG = "Watchdog"
+            REBOOT_CAUSE_HARDWARE_OTHER = "Hardware - Other"
+            REBOOT_CAUSE_NON_HARDWARE = "Non-Hardware"
```

Comment:
These constants are defined in the ChassisBase class you should not need to redefine them here, this class should already have them via inheritence.

Reply:
ok, I got it,
I stated in Docstrings to list the values that can be returned.

Reply:
Oh, I missed that these are inside the docstring. Sorry!